README.md

```diff
@@ -52,32 +52,32 @@ end
 ```
 
 ### Configure it
-Set these mandatory options in the rails_helper:
+Set these optiional options in the rails_helper:
 
 | Option | Value | Description |
 | -- | -- | -- |
-| body_file_path | Pathname instance or fullpath string | Json file that will include openapi version, the basic information about the api, api title and api version. |
-| desc_folder_path | Pathname instance or fullpath string | Folder with markdown descriptions. |
+| descriptions_location | Pathname instance or fullpath string | Folder with markdown descriptions. |
 | schema_request_folder_path | Pathname instance or fullpath string | Folder with request schemas of resources. |
 | schema_response_folder_path | Pathname instance or fullpath string | Folder with response schemas of resources. |
 | schema_response_fail_file_path | Pathname instance or fullpath string | Json file that with the default schema of a failed response. |
-
-Optional settings:
-
-| Option | Value| Description |
-| -- | -- | -- |
+| openapi_version | string | Openapi version (default: '3.0.0' ) |
+| api_version | string | Api Version (default: '1.0') |
+| title | string | Documentation title (default: 'API Documentation') |
+| description | Pathname instance or fullpath string or string | Description of the documentation (default: '') |
```

Comment:
What description is this? What usually goes here?

Reply:
description at the top of the documentation 
![API Example 2020-08-07 15-37-49](https://user-images.githubusercontent.com/973872/89651298-022e4d80-d8c4-11ea-84b3-0baef2e790f4.png)